internal/command/windows.go

```diff
@@ -223,14 +236,115 @@ func (cmd *windows) setupContainerImages(flags *windowsFlags, args []string) err
 		switch arch {
 		case ArchAmd64:
 			image.SetEnv("GOARCH", "amd64")
-			image.SetEnv("CC", "x86_64-w64-mingw32-gcc")
+			image.SetEnv("CC", "zig cc -target x86_64-windows-gnu -Wdeprecated-non-prototype")
 		case Arch386:
 			image.SetEnv("GOARCH", "386")
-			image.SetEnv("CC", "i686-w64-mingw32-gcc")
+			image.SetEnv("CC", "zig cc -target x86-windows-gnu -Wdeprecated-non-prototype")
 		}
 
 		cmd.Images = append(cmd.Images, image)
 	}
 
 	return nil
 }
+
+// makeWindowsResource create a windows resource under the project root
+// that will be automatically linked by compliler during the build
+func makeWindowsResource(ctx Context, image containerImage) (string, error) {
```

Comment:
Would it be actually possible to have this part of the image as one of the fyne-cross tool? This would allow to run this in the container and have it working with k8s.

Reply:
Maybe I'm missing something but this should not affect the previous implementation.
`makeWindowsResource` is replacing the `WindowsResource` from the common.go file.
The main difference it is that now the windows resource creation is done using a Go library instead of the `windres` CLI tool. 

Note: I still need to add the `internal/cmd/fyne-cross-s3` to the new base image.

Reply:
Indeed, this is not a regression as this path is not working with the k8s port. I am actually wondering if we were to use fyne build if that would solve this problem. I can look into switching to use fyne build.